ambry-rest/src/main/java/com.github.ambry.rest/NettyServerFactory.java

```diff
@@ -47,17 +52,24 @@
    * @param sslFactory the {@link SSLFactory} used to construct the {@link javax.net.ssl.SSLEngine} used for handling
    *                   SSL requests.
    * @throws IllegalArgumentException if any of the arguments are null.
+   * @throws ReflectiveOperationException if a netty-specific {@link SSLFactory} cannot be instantiated via reflection.
    */
   public NettyServerFactory(VerifiableProperties verifiableProperties, MetricRegistry metricRegistry,
       final RestRequestHandler requestHandler, final PublicAccessLogger publicAccessLogger,
-      final RestServerState restServerState, SSLFactory sslFactory) {
+      final RestServerState restServerState, SSLFactory sslFactory) throws ReflectiveOperationException {
```

Comment:
Since we're declaring that here, why not also change Utils.getObj() to throw ReflectiveOperationException instead of the five it currently declares?  I wasn't aware until now of this base class that was apparently added in JDK 1.7.

Reply:
Changed the various `getObj` methods to throw this exception.

Reply:
Did you check that in?  Or will that be in a followup?

Reply:
Checked in now. Sorry, got interrupted in between writing these comments and git pushing